Lib/test/support/__init__.py

```diff
@@ -785,6 +785,7 @@ def gc_collect():
     gc.collect()
     gc.collect()
     gc.collect()
+    gc._collect_delayed_objects()
```

Comment:
```suggestion
    gc._collect_delayed_objects()

```

Will apply after build bot is passed.